src/SFA.DAS.LevyTransferMatching.Web/Models/Cache/CreatePledgeCacheItem.cs

```diff
@@ -25,5 +25,6 @@ public CreatePledgeCacheItem(Guid key)
         public List<string> JobRoles { get; set; }
         public List<string> Levels { get; set; }
         public List<string> Locations { get; set; }
+        public IDictionary<int, IEnumerable<string>> MultipleValidLocations { get; set; }
```

Comment:
I don't think we should pollute this cache item with a property to hold this. Nothing against using the cache for this purpose (can't think of a viable alternative really), but I think a new cache item would be better. (See also, comment about a back link bug)

Reply:
This has been split out into a `LocationSelectionCacheItem`, accessed via the same key with a prefix of the same name (e.g. `"LocationSelectionCacheItem_BE022A26-5568-4CC2-B459-226B7FE6E0C1"`), as discussed.